specs/validator-system-v2/asset-manager.md

```diff
@@ -0,0 +1,218 @@
+# Asset Manager
+
+<!-- All glossary references in this file. -->
+
+[g-l2-output]: ../glossary.md#l2-output-root
+[g-validator]: ../glossary.md#validator
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+- [Composition of Asset Manager](#composition-of-asset-manager)
+- [Undelegation Delay of 7 days](#undelegation-delay-of-7-days)
+- [Calculation of Slashing Amount](#calculation-of-slashing-amount)
+- [Difference from ERC-4626](#difference-from-erc-4626)
+  - [Non-transferable Shares](#non-transferable-shares)
+  - [Global Vault Structure](#global-vault-structure)
+- [Security Considerations](#security-considerations)
+  - [Virtual Offset](#virtual-offset)
+  - [Virtual Share](#virtual-share)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview
+
+The Asset Manager oversees the delegation and undelegation of KRO tokens and Kroma Guardian House (KGH) NFTs.
+It calculates and distributes rewards based on [output root][g-l2-output] submissions and manages slashing penalties.
+Token holders deposit their assets into the system and receive rewards upon undelegation. The Asset Manager supervises
+the assets of all [validators][g-validator] and delegators within the Kroma validator system.
+
+## Composition of Asset Manager
+
+Asset Manager incorporates a `Vault` component, inspired by the ERC-4626 standard. Its responsibilities include:
+
+1. Management of delegated KRO and KGH NFTs
+
+    - Adopts the structure of ERC-4626 to issue shares when KRO and KGH NFTs are delegated.
+
+2. Slashing and Reward Calculation
+
+    - Calculates rewards for delegators based on the output root submission and manages slashing of assets when
+   challenges occur.
+
+Each validator registered in `ValidatorManager` has a corresponding `Vault` struct. The `Vault` struct is defined as
```

Comment:
Don't forget to link to the `ValidatorManager`.

Reply:
Adding this here will break the compile of serving this book, so @seolaoh could you please add this on the following PR, where validator manager is?